.github/actions/container-scanning/action.yml

```diff
@@ -48,7 +48,7 @@ runs:
         role-duration-seconds: 1800
         role-session-name: github-actions-inspector
     - name: Scan built image with Inspector
-      uses: aws-actions/vulnerability-scan-github-action-for-amazon-inspector@v1.1.2
+      uses: aws-actions/vulnerability-scan-github-action-for-amazon-inspector@v1
```

Comment:
https://github.com/aws-actions/vulnerability-scan-github-action-for-amazon-inspector/issues/111#issuecomment-2794204653